test/pummel/test-stream-pipe-multi.js

```diff
@@ -66,8 +66,11 @@ FakeStream.prototype.close = function() {
 
 // expect all streams to close properly.
 process.on('exit', function() {
-  assert.strictEqual(cnt, wclosed, 'writable streams closed');
-  assert.strictEqual(cnt, rclosed, 'readable streams closed');
+  wclosed = rclosed = 0;
```

Comment:
I am not sure why this was added here. In general I think it would be best to just remove the error message in the 
 strictEqual`

Reply:
Agreed, @johenry if you could push a commit that just makes it

```js
assert.strictEqual(cnt, wclosed)
assert.strictEqual(cnt, rclosed)
```

that'd be great!

Reply:
@gibfahn Done. Thanks for the help.